app/cdap/components/RecipeList/ViewAllRecipes.tsx

```diff
@@ -14,24 +14,28 @@
  * the License.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import T from 'i18n-react';
 import RecipeList from 'components/RecipeList';
-
 import Box from '@material-ui/core/Box';
 import { SortBy, SortOrder } from './types';
 import Breadcrumbs from '@material-ui/core/Breadcrumbs';
 import { Link } from 'react-router-dom';
 import Typography from '@material-ui/core/Typography';
 import { getCurrentNamespace } from 'services/NamespaceStore';
+import RecipeDetails from 'components/RecipeDetails';
+import { IRecipeItem } from 'components/RecipeDetails';
 
 const PREFIX = 'features.WranglerNewUI.Recipe';
 
 const redirectToObj = `/ns/${getCurrentNamespace()}/wrangle`;
 
 const ViewAllRecipies = () => {
+  const [showRecipeDetails, setShowRecipedetails] = useState(false);
```

Comment:
is this referring to the Panel or drawer then plz name accordingly
showRecipesPanel or showRecipesDrawer
also, plz keep the names consistent with what we already used
if anywhere we missed or used wrong naming, plz go to respective feature pr and add comments in that pr
this will help us reduce the issues considerably